index.html

```diff
@@ -217,14 +217,14 @@ <h2 class="view">I'm always<br>
             so if you'd like to<br>
             chat please get in
         </h2>
-        <form>
-            <input type="text" class="input" placeholder="Full name">
+        <form action="https://formspree.io/f/xzbodvoe" method="post">
+            <input type="text" class="input" placeholder="Full name" required maxlength="30">
             <hr>
-            <input type="text" class="input" placeholder="Email address">
+            <input type="email" class="input" placeholder="Email address" required>
             <hr>
-            <input type="text" class="box" placeholder="Enter text here">
+            <textarea class="box" placeholder="Enter text here" required maxlength="500"></textarea>
+            <button type="button" class="touch">Get in touch</button>
```

Comment:
- [ ] You have done a great job by implementing the requested changes. But we still cannot submit the form. Kindly change the type of the button from `button` to `submit`.
- [ ] Please before request another review, make sure your form is submitted successfully. You should see this message below after submitting the form:
<img width="275" alt="image" src="https://user-images.githubusercontent.com/77190203/150020550-08a6d654-7eea-49d3-a31e-c44e950d8fda.png">